docs/markdown/snippets/structured_sources.md

```diff
@@ -0,0 +1,25 @@
+## structured_sources()
+
+A new function, `structured_sources()` has been added. This function allows
+languages like Rust and Cython which depend on the filesystem layout at compile
```

Comment:
May want to remove any usage of Cython from this PR since I think you took it out :(.